tika-core/src/test/java/org/apache/tika/language/LanguageIdentifierTest.java

```diff
@@ -103,10 +103,45 @@ public void testClearAddAndInitProfiles() throws IOException {
         identifier = new LanguageIdentifier(deProfile);
         assertEquals("de", identifier.getLanguage());
         assertTrue(identifier.isReasonablyCertain());
-  }
+    }
```

Comment:
True, although my reformatting followed the indentation level. I have changed it back.